### PR TITLE
Adding the "VK_KHR_multiview"-extension.

### PR DESCRIPTION
**Pull request by jpryne**
Adding the "VK_KHR_multiview"-extension. This may well require additional tags in order to function properly. I will test & revise.

* [ ] Added an entry to `CHANGELOG_VULKANO.md` or `CHANGELOG_VK_SYS.md` if knowledge of this change could be valuable to users
* [ ] Updated documentation to reflect any user-facing changes - in this repository
* [ ] Updated documentation to reflect any user-facing changes - PR to the [guide](https://github.com/vulkano-rs/vulkano-www) that fixes existing documentation invalidated by this PR.

### CODE REVIEW

**Comment by jpryne** [comment]
Just discovered I also need to add "VK_KHR_get_physical_device_properties2" to the extensions list. Will follow this up with another PR once this is merged.

File: vulkano/src/device/extensions.rs [hunk 1]
```diff
@@ -114,6 +114,7 @@ device_extensions! {
     khr_dedicated_allocation => b"VK_KHR_dedicated_allocation",
     khr_incremental_present => b"VK_KHR_incremental_present",
     ext_debug_marker => b"VK_EXT_debug_marker",
+    khr_multiview => b"VK_KHR_multiview",
```
Comment (jpryne) [inline vulkano/src/device/extensions.rs]: khr_get_physical_device_properties2 => b"VK_KHR_get_physical_device_properties2",

Comment (jpryne) [reply vulkano/src/device/extensions.rs]: Ack, never mind. That's an instance extension, & it's already in place.

**Comment by AustinJ235** [comment]
I'll merge if you are ready once https://github.com/vulkano-rs/vulkano/pull/1244 goes through. This is preventing merges due to the CI requirement.

**Comment by AustinJ235** [comment]
@jpryne Go ahead and pull in master again and should be ready to go.

**Pull request merged**